nautobot/extras/forms/mixins.py

```diff
@@ -347,22 +347,22 @@ def save_relationships(self, *, instance, nullified_fields):
     def clean(self):
         # Get any initial required relationship objects errors (i.e. non-existent required objects)
         required_objects_errors = self.model.required_related_objects_errors(output_for="ui")
-        already_invalidated_slugs = []
+        already_invalidated_keys = []
         for field, errors in required_objects_errors.items():
             self.add_error(None, errors)
-            relationship_slug = field.split("__")[0][3:]
-            already_invalidated_slugs.append(relationship_slug)
+            relationship_key = field.split("__")[0][3:]
```

Comment:
Not blocking for this PR but we should make this smarter, it'll probably break if the relationship key itself contains `__`.